python/cudf/cudf/tests/test_config.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+import pytest
+
+import cudf
+
+
+@pytest.fixture(scope="module")
+def configuration_demo():
```

Comment:
This name should reflect the name of the configuration, like `register_odd_config`, rather than `configuration_demo` which sounds unrelated.